app-devel/git-absorb/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=git-absorb
+PKGDES="Automatically write fixup! commits, similar to hg absort"
+PKGSEC=devel
+PKGDEP="git"
+BUILDDEP="rustc llvm libgit2"
+
+CARGO_AFTER="--features=git2/vendored-libgit2"
+USECLANG=1
+# FIXME: loongson3 + lld building issue
+NOLTO=1
```

Comment:
Then we only need to disable LTO for loongson3. Use `NOLTO__LOONGSON3=1` instead.